File: WPServer/WPProjectServer/WPProjectServer/Logic/SubsystemLogic/DataPullSubLogic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WPProjectServer.DatabaseObjects;

namespace WPProjectServer.Logic.SubsystemLogic
{
    public class DataPullSubLogic : ISubsystemLogic
    {
        public CopyStatus GetCopyStatus(Copy copy)
        {
            return new CopyStatus();
        }

        public Copy GetCopy(string code)
        {
            return new Copy();
        }

        public Copy[] GetAllCopies(Book book)
        {
            return new Copy[0];
        }

        public User GetUser(string code)
        {
            return new User();
        }

        public Book GetBook(string Code)
        {
            return new Book();
        }

        public Book[] GetBooks(Author author)
        {
            return new Book[0];
        }

        public Book[] GetBooks(string title)
        {
            return new Book[0];
        }


    }
}

File: WPWeb/Lib/src/main/java/controllers/SubLogic/DataPullSubLogic.java
package src.main.java.controllers.SubLogic;

import src.main.java.controllers.LogicFactory;
import src.main.java.models.*;

/**
 * Created by Szymon on 2015-10-13.
 */
public class DataPullSubLogic extends ISubsystemLogic
{
        public DataPullSubLogic(LogicFactory lgF)
        {
                super(lgF);
        }

        public CopyStatus GetCopyStatus(Copy copy)
        {
                return new CopyStatus();
        }

        public Copy GetCopy(String code)
        {
                return new Copy();
        }

        public Copy[] GetAllCopies(Book book)
        {
                return new Copy[0];
        }

        public User GetUser(String code)
        {
                return new User();
        }

        public Book GetBook(String Code)
        {
                return new Book();
        }

        public Book[] GetBooks(Author author)
        {
                return new Book[0];
        }

        public Book[] GetBooks(String title)
        {
                return new Book[0];
        }


}
